### PR TITLE
Exclude instrumentation tests from generic build patterns

### PR DESCRIPTION
**Pull request by BenHenning**
## Explanation

This change automatically excludes all instrumentation tests from generic patterns like ``bazel build //...`` since instrumentation tests don't build correctly yet.

Reference Bazel documentation for the manual tag: https://docs.bazel.build/versions/main/guide.html#specifying-targets-to-build.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: instrumentation/oppia_instrumentation_test.bzl [hunk 1]
```diff
@@ -32,9 +32,10 @@ def oppia_instrumentation_test(
         deps = [":%s_lib" % name],
     )
 
-    # TODO(#3617): Target isn't supported yet.
+    # TODO(#3617): Target isn't supported yet. Remove the manual tag once fixed.
     native.android_instrumentation_test(
         name = name,
         target_device = "@android_test_support//tools/android/emulated_devices/generic_phone:android_23_x86_qemu2",
         test_app = ":%sBinary" % name,
+        tags = ["manual"],
```
Comment (FareesHussain) [inline instrumentation/oppia_instrumentation_test.bzl:40]: Actually, I was wondering how to ignore test targets and this was exactly what I was looking for

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

File: instrumentation/oppia_instrumentation_test.bzl (hunk 1 above)
Comment (BenHenning) [reply instrumentation/oppia_instrumentation_test.bzl:40]: I didn't think about there being a tag until today; we probably can revert the ComputeAffectedTests bits but eh.

**Comment by BenHenning** [comment]
Thanks! Enabling auto-merge.

**Pull request merged**